homeassistant/components/shelly/const.py

```diff
@@ -35,6 +35,7 @@
 LOGGER: Logger = getLogger(__package__)
 
 CONF_COAP_PORT: Final = "coap_port"
+CONF_SCRIPT: Final = "script"
```

Comment:
I don't think `script` tells you exactly what it's about. Maybe `scripts_support` would be better?